tests/e2e_operations.py

```diff
@@ -150,6 +150,55 @@ def test_forced_snapshot(network, args):
     raise RuntimeError("Could not find matching snapshot file")
 
 
+# https://github.com/microsoft/CCF/issues/1858
+@reqs.description("Generate snapshot larger than ring buffer max message size")
+def test_large_snapshot(network, args):
+    primary, _ = network.find_primary()
+
+    # Submit some dummy transactions
+    entry_size = 1000  # Lower bound on serialised write set size
+    iterations = int(args.max_msg_size_bytes) // entry_size
```

Comment:
How many iterations is this in practice? We could easily end up sending 100s of requests here, resulting in a pretty slow test. If so, we should find a more efficient way to build a large snapshot (write-multiplying endpoint).

Reply:
This was about 100 iterations, which took about 4 seconds in practice. I've bumped `entry_size` to `10000`, which reduces it to 10 iterations, or about half a second.